web/src/drafts.js

```diff
@@ -274,6 +275,62 @@ export function update_draft(opts = {}) {
 
 export const DRAFT_LIFETIME = 30;
 
+export function get_draft_ids_for_topic(stream_id, topic) {
+    const drafts = draft_model.get();
+    return Object.keys(drafts)
+        .filter(
+            (draft_id) =>
+                drafts[draft_id].type === "stream" &&
+                drafts[draft_id].stream_id === stream_id &&
+                drafts[draft_id].topic === topic,
+        )
+        .sort((draft_a, draft_b) => drafts[draft_a].updatedAt - drafts[draft_b].updatedAt);
+}
```

Comment:
This is a case-sensitive topic comparison; should this be case-insensitive, perhaps using `util.same_topic`?